app/Hutch.Relay/Models/RelaySubTaskModel.cs

```diff
@@ -0,0 +1,11 @@
+using Hutch.Relay.Data.Entities;
```

Comment:
Why?

Reply:
I had a missing model - `RelaySubTask` needed `SubNodeModel`, this has been added.